src/header.jsx

```diff
@@ -5,26 +5,34 @@ import { Nav } from "./nav.js";
 
 function Header() {
   return (
-    <jspm-package-header>
-      <header>
-        <Logo />
-        <Search />
+    <div>
```

Comment:
Use of custom element syntax is intentional here.

Reply:
Yes, but that's making hard to write media-queries. They don't inherit fields. And instead overwriting them 😅

Reply:
If there is any specific usage for this, i will wrap this with those tags